iothub/device/src/Message.cs

```diff
@@ -362,6 +362,21 @@ public DateTimeT CreationTimeUtc
             }
         }
 
+        /// <summary>
+        /// True if the message is set as a security message
+        /// </summary>
+        public bool IsSecurityMessage
+        {
+            get
+            {
+#if NETMF
+                return CommonConstants.SecurityMessageInterfaceId.Equals(this.GetSystemProperty(MessageSystemPropertyNames.InterfaceId));
```

Comment:
We should not have any NETMF new code.

Reply:
New features are allowed on netstandard2.0 only.

Reply:
As long as we have NETMF running @ the gate we can't remove that code.

---
In reply to: [301723046](https://github.com/Azure/azure-iot-sdk-csharp/pull/999#discussion_r301723046) [](ancestors = 301723046)